compiler/src/iree/compiler/Codegen/LLVMCPU/BUILD.bazel

```diff
@@ -67,6 +67,7 @@ iree_compiler_cc_library(
         "LLVMCPUSynchronizeSymbolVisibility.cpp",
         "LLVMCPUTile.cpp",
         "LLVMCPUTileAndFuse.cpp",
+        "LLVMCPUTileRootAndFuseProdCons.cpp",
```

Comment:
I dont see much value in trying to save number of characters of a file. Make it explicit.